packages/sdk/src/sync-agent/spaces/models/channel.ts

```diff
@@ -67,19 +75,31 @@ export class Channel extends PersistedObservable<ChannelModel> {
         }
     }
 
+    /** Joins the channel. */
     async join() {
         const channelId = this.data.id
         await this.riverConnection.call(async (client) => {
             await client.joinStream(channelId)
         })
     }
 
+    /** Sends a message to the channel.
+     * @param message - The message to send.
+     * @param options - Additional options for the message.
+     * @returns The event id of the message.
+     */
     async sendMessage(
         message: string,
         options?: {
+            /** If set, this message will be linked to a thread with the specified message. */
             threadId?: string
+            /** If set, this message will be linked as a reply to the specified message. */
             replyId?: string
+            // TODO: would be great to improve the usability here. Its not that ergonomic to pass the `ChannelMessage_Post_Mention` payload itself
+            // and its a bit unclear about what are the intetion of the fields.
+            /** The users that are mentioned in the message */
```

Comment:
> its a bit unclear about what are the intention of the fields

For example, to create a `ChannelMessage_Post_Mention` you can pass `userId`, `displayName` and type `atChannel` or `atRole` or leave it empty for user mentions

It's not clear for me how to mention a specific channel. e.g: mention `atChannel`. But which channel? There's no `channelId` field. 

For the role mention, `atRole` mention behavior proto expects a `ChannelMessage_Post_RoleMention` which holds the `roleId`

Reply:
atRoles aren't currently implemented
we should just accept the PlainMessage<of the proto> and call it a day